o-fish-ios/ViewModel/Report/DeliveryViewModel.swift

```diff
@@ -47,5 +47,5 @@ class DeliveryViewModel: ObservableObject, Identifiable {
         return delivery
     }
 
-    var isEmtpy: Bool { date == nil && location.isEmpty && business.isEmpty }
+    var isEmpty: Bool { date == nil && location.isEmpty && business.isEmpty }
```

Comment:
what actually changed?

Reply:
mistype. was isEm"t"py, should "isEmpty"